utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -674,7 +674,8 @@ function ConvertTo-FormattedJSONParameterObject {
         '}'
     ) | Out-String
 
-    # [2.2] Syntax: All single-quotes are double-quotes
+    # [2.2] Syntax: All double quotes must be escaped & single-quotes are double-quotes
```

Comment:
@AlexanderSehr  Does this change impact other modules docos?

Reply:
It was already in main and I just merged it - and refreshed the docs